Jenkinsfile_CNP

```diff
@@ -70,7 +70,8 @@ withPipeline(type, product, app) {
 
         def githubApi = new GithubAPI(this)
         if (!githubApi.getLabelsbyPattern(env.BRANCH_NAME, "keep-helm")) {
-            enableCleanupOfHelmReleaseAlways()
+            //commenting as enableCleanupOfHelmReleaseOnSuccess is now the default
+            //enableCleanupOfHelmReleaseAlways()
```

Comment:
Doesn't the inverse now need to be handled if the default has changed? Or is this handled somewhere else?